ci/build-all-steps.yml

```diff
@@ -72,3 +72,17 @@ steps:
   - powershell: ./ci/stop-collect-diagnostics.ps1
     displayName: Stop collect diagnostics
     condition: and(always(), eq(variables.collect_diagnostics, 'true'), ne(variables['numTasks'], 0))
+
+  - powershell: |
+      # Build the release branch name
+      $startdate = Get-Date -Date '2020-01-20'
```

Comment:
it seems weird to have a hardcoded date here

Reply:
Yeah - that was kinda arbitary, I don't know of a better way to calculate sprint though

Reply:
Will comment below